integration/src/test/java/dev/ikm/tinkar/integration/search/SearcherIT.java

```diff
@@ -121,4 +126,26 @@ public void searchFromDescendantsOfConceptWithCustomCalculatorIT() throws Except
         // 2) Refrenced component type restriction
         assertEquals(6, searchResults.size(), "Exactly 6 search results should be returned");
     }
+
+    @Test
+    public void searchConceptsWithMembershipSemantic() throws Exception {
+        // test memberPatternId does not exist
+        EntityProxy.Concept conceptProxy = EntityProxy.Concept.make(PublicIds.newRandom());
+        List<PublicId> conceptIds = Searcher.membersOf(conceptProxy.publicId());
+        assertTrue(conceptIds.isEmpty(), "memberPatternId does not exist, should return empty list");
```

Comment:
Could each of these assertions be separate tests? I know these aren't unit tests, so they are more costly to run, so if it loads up up a DB for each test, probably best to leave them all together.

Reply:
@stenkarl, great suggestion! @dettmoney the database is started in the BeforeAll method so we should be able to split these tests up without the performance hit of starting / stopping the database each time.